### PR TITLE
add -long option to fa2twobit

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: data_managers/data_manager_twobit_builder/data_manager/twobit_builder.py [hunk 1]
```diff
@@ -71,6 +74,8 @@ def main():
     parser.add_option('-f', '--fasta_filename', dest='fasta_filename', action='store', type="string", default=None, help='fasta_filename')
     parser.add_option('-d', '--fasta_dbkey', dest='fasta_dbkey', action='store', type="string", default=None, help='fasta_dbkey')
     parser.add_option('-t', '--fasta_description', dest='fasta_description', action='store', type="string", default=None, help='fasta_description')
+    parser.add_option('-l', '--long', dest='long', action="store_true", default=False, help='For big genomes you need to pass the -long option.')
```
Comment (bernt-matthias) [inline data_managers/data_manager_twobit_builder/data_manager/twobit_builder.py:77]: Could not find any docs that `-long` exists for `fato2bit`? Wondering what is "big"? Can we set it automatically based on the file size?

Comment (bgruening) [reply data_managers/data_manager_twobit_builder/data_manager/twobit_builder.py:77]: The long option was given in the error message. I also did not find in the documentation :( 
So no idea. With the option, it worked.

File: data_managers/data_manager_twobit_builder/data_manager/twobit_builder.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="twobit_builder_data_manager" name="TwoBit" tool_type="manage_data" version="0.0.4" profile="19.05">
+<tool id="twobit_builder_data_manager" name="TwoBit" tool_type="manage_data" version="0.0.5" profile="19.05">
```
Comment (bernt-matthias) [inline data_managers/data_manager_twobit_builder/data_manager/twobit_builder.xml:1]: Do we want to use `377` as version?

**Review by bernt-matthias — COMMENTED** [review]
From the sources:

```
  "   -long          use 64-bit offsets for index.   Allow for twoBit to contain more than 4Gb of sequence. \n"
  "                  NOT COMPATIBLE WITH OLDER CODE.\n"
```
So this leaves two questions to me: 

1. will data table entries generated with `--long` be usable by older tools? Otherwise we might want to consider to version the tool data table entries, or?
2. Should we just use `--long` always?

**Comment by bgruening** [comment]
> So this leaves two questions to me:
>  *   will data table entries generated with --long be usable by older tools? Otherwise we might want to consider to version the tool data table entries, or?
>  *  Should we just use --long always?

I have no idea :(

**Comment by bernt-matthias** [comment]
Then it seems reasonable to leave it as optional argument for now and collect some experience with it... 

Anything else you want to change here?

**Comment by bernt-matthias** [comment]
Can you remove the version here: https://github.com/galaxyproject/tools-iuc/blob/main/data_managers/data_manager_twobit_builder/data_manager_conf.xml xref https://github.com/galaxyproject/galaxy/pull/12255 (maybe we should just do this for all DMs in one push .. otherwise it will be copy pasted for years)

**Comment by bgruening** [comment]
No, please merge or adopt as you like.

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**